frontend/src/components/Meal/MealContainer.js

```diff
@@ -1,29 +1,27 @@
-import React from 'react';
-import { Meal } from "./Meal";
-import { MealForm } from "./MealForm";
-import { makeStyles } from '@material-ui/core/styles';
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Day } from "./Day";
 
 const useStyles = makeStyles({
-    mealContainer: {
-      padding: 30,
-    },
-  });
+  mealContainer: {
+    padding: 30,
+  },
+});
 
 const MealContainer = () => {
-    const classes = useStyles();
+  const classes = useStyles();
+  const [daysOffset, setDaysOffset] = React.useState([0,1,2,3,4,5,6]);
```

Comment:
`[0,1,2,3,4,5,6]` -> please move it to config, same place where APP_URL is.

Reply:
done